records/validation.go

```diff
@@ -0,0 +1,49 @@
+package records
+
+import (
+	"fmt"
+	"net"
+)
+
+// validateMasters checks that each master in the list is a properly formatted host:ip pair.
+// duplicate masters in the list are not allowed.
+// returns nil if the masters list if empty, or else all masters in the list are valid.
```

Comment:
no, because config.Masters functions as "fallback masters" if the current leading master can't be detected from zk. If zk is not configured AND masters == nil, then the config analyzer should error out -- but we don't handle that here.